doc/api.md

```diff
@@ -327,60 +335,60 @@ NOTE: The devices object has the same format as the getAvailableDevices result f
 changes. The listener will receive an object with the following structure:
 ```javascript
 {
-"id": id, // the id of the participant that changed his email
-"email": email // the new email
+    'id': id, // the id of the participant that changed his email
+    'email': email // the new email
 }
 ```
 * **filmstripDisplayChanged** - event notifications about the visibility of the filmstrip being updated.
 ```javascript
 {
-"visible": visible, // Whether or not the filmstrip is displayed or hidden.
+    'visible': visible // Whether or not the filmstrip is displayed or hidden.
```

Comment:
You could be really strict about following coding styling and use property shorthands.

Reply:
Here we're providing example of the structure. Isn't the shorthand gonna be a little confusing?

How about maybe renaming the `visible` value variable to something else? 

Or maybe to change it to:
```
{
    visible: boolean
}
```

WDYT?

Reply:
Better

Reply:
Fixed!